applications/spring-shell/src/test/java/org/springframework/sbm/MigrateSimpleMuleAppDataweaveIntegrationTest.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright 2021 - 2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.sbm;
+
+import org.junit.jupiter.api.*;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.client.RestTemplate;
+
+import java.util.*;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@TestMethodOrder(MethodOrderer.MethodName.class)
+//@Disabled("Temporary disabled before CI will be fixed with docker in docker issue: #351")
+class MigrateSimpleMuleAppDataweaveIntegrationTest extends IntegrationTestBaseClass {
+    private final RestTemplate restTemplate = new RestTemplate();
+    private static RunningNetworkedContainer tmDataweaveContainer;
+
+    @Override
+    protected String getTestSubDir() {
+
+        return "mule-app/spring-dw-mule";
+
+    }
+
+    @BeforeAll
+    public static void beforeAll() {
+        IntegrationTestBaseClass.beforeAll();
+
+        // Will need to ensure this is set globally for the test
+        System.setProperty("sbm.muleTriggerMeshTransformEnabled", "true");
```

Comment:
I looked into overriding `@SpringProperties`, but this would not get picked up by the transformer that would swap out the stub code for the code that will send the transformation out to the TriggerMesh Dataweave transformer. If there's a cleaner way to do this, then would appreciate hearing it.

Reply:
I think we should rework this approach and set the flag in the action of a dedicated recipe, e.g. `migrate-mule-to-boot-tm` and then just call it from here.